### PR TITLE
Improvements: Add Url type to tabs configuration

### PR DESCRIPTION
**Pull request by lciolecki**
* Provided new tab type => External, just url to external page
* Allow to open tab in external window / tab / etc.. (add target option)

### CODE REVIEW

File: ui/server/src/main/resources/defaultUiConfig.conf [hunk 1]
```diff
@@ -185,6 +185,7 @@ componentsAppUrl: ${?NU_FE_COMPONENTS_URL}
 tabs: [
   {title: "Scenarios", type: "Local", url: "/processes", id: "scenarios"},
   {title: "Components", type: "Remote", url: "nu_components/list@"${componentsAppUrl}, id: "components"}
+  {title: "Docs", type: "External", url: "https://nussknacker.io/documentation/", id: "docs", target: "_blank"}
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/server/src/main/resources/defaultUiConfig.conf:188]: Hmm, but there can be situation when you want to do external url in the same browser. But you're right, other cases are not so important..

[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/server/src/main/resources/defaultUiConfig.conf]: target was removed.

**Review by JulianWielga — APPROVED** [review]

File: docs/installation_configuration_guide/DesignerConfiguration.md [hunk 2]
```diff
@@ -598,19 +598,21 @@ Tabs (in main menu bar, such as Scenarios etc.) can be configured in the followi
...
-| requiredPermission | string              | Optional parameter, name of [Global Permission](#security) |
-                                       
+| Parameter name     | Type                         | Description                                                |
+| --------------     |------------------------------| -----------                                                |
+| id                 | string                       | Unique identifier                                          |
+| title              | string                       | Title appearing in UI                                      |
+| type               | IFrame/Local/Remote/Url | Type of tab (see below for explanation)                    |
+| url                | string                       | URL of the tab                                             |
+| requiredPermission | string                       | Optional parameter, name of [Global Permission](#security) |
+| target             | string                       | Optional parameter, html a [target param](https://www.w3schools.com/tags/att_a_target.asp) |
```
Comment (dswiecki) [inline docs/installation_configuration_guide/DesignerConfiguration.md:608]: remove target from here

**Comment by dswiecki** [comment]
you can add entry in change log because its very nice feature 👍

File: ui/client/components/MenuBar.tsx [hunk 3]
```diff
@@ -24,15 +25,27 @@ function useStateWithRevertTimeout<T>(startValue: T, time = 10000): [T, React.Di
   return [value, setValue]
 }
 
+function MenuElement(props: {tab: DynamicTabData}): JSX.Element {
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/client/components/MenuBar.tsx]: Gracias amigo! I thought, I had removed it.

File: docs/installation_configuration_guide/DesignerConfiguration.md (hunk 2 above)
Comment (lciolecki) [reply docs/installation_configuration_guide/DesignerConfiguration.md:608]: Gracias amigo! I thought, I had removed it.

**Pull request merged**